pkg/network/sriov/sriov_test.go

```diff
@@ -121,7 +128,7 @@ var _ = Describe("SRIOV", func() {
     }
   }
 }
-]`
+]`, booVMNetworkPodIfaceName, fooVMNetworkPodIfaceName)
```

Comment:
You replace the order here (compared to the prev config).
If tests pass, I guess it is fine. Just a bit confusing when reviewing.